src/test/java/lab5/players/AIPlayer2Test.java

```diff
@@ -0,0 +1,50 @@
+package lab5.players;
+
+import lab5.game.Board;
+import lab5.game.Position;
+import lab5.game.Row;
+import lab5.game.Col;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+class AIPlayer2Test {
+    private AIPlayer2 aiPlayer;
+    private Board board;
+
+    @BeforeEach
+    void setUp() {
+        aiPlayer = new AIPlayer2();
+        board = new Board();
+    }
+
+    @Test
+    void testPickNextMove_PrioritizesCenter() {
+        Position move = aiPlayer.pickNextMove(board);
+        assertEquals(new Position(Row.Middle, Col.Middle), move, "AI should prioritize center position");
+    }
+
+    @Test
+    void testPickNextMove_FirstAvailableMove() {
+        // Mark center as occupied
+        board.placeNextToken(new Position(Row.Middle, Col.Middle));
+
+        Position move = aiPlayer.pickNextMove(board);
+
+        assertNotNull(move, "AI should pick a valid move");
+        assertTrue(board.getEmptyCells().contains(move), "Move should be a valid empty position");
+    }
```

Comment:
This should test that the top middle spot gets picked next.

Reply:
You should fully test that the each of the correct sequence of spots gets picked.  See the sample solution.